plugin.go

```diff
@@ -2,31 +2,32 @@
 package plasmactlbump
 
 import (
```

Comment:
- [ ] Throw an error during propagation when domain repo contains unversioned changes in a resource
      echo " " >> interaction/softwares/roles/rainloop/templates/Dockerfile.j2
      compose and sync
```
WARNING  resource interaction__softwares__odoo is marked as updated, but version in build (def5f0ef5abd5) and artifact (def5f0ef5abd5) are identical, skipping.
> 
ERROR  resource interaction__softwares__odoo has unversioned changes. You need to commit these changes.
```